homeassistant/components/amberelectric/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "domain": "amberelectric",
+    "name": "Amber Electric",
+    "documentation": "https://www.home-assistant.io/integrations/amberelectric",
+    "config_flow": true,
+    "codeowners": [
+        "@madpilot"
+    ],
+    "requirements": [
+        "amberelectric==1.0.0"
```

Comment:
What's difference between this package and https://pypi.org/project/amberelectric.py/ ?

Reply:
`amberelectric.py` got renamed `amberelectric`. The latter is the official version.